sklearn/preprocessing/tests/test_data.py

```diff
@@ -449,6 +463,11 @@ def test_warning_scaling_integers():
     clean_warning_registry()
     with warnings.catch_warnings(record=True):
         warnings.simplefilter("always")
+        assert_warns(UserWarning, scale, X)
```

Comment:
Please use the `assert_warns_message` utility instead:

```python
sklearn.utils.testing import assert_warns_message
```

`git grep "assert_warns_message("` for sample usage in the scikit-learn code base.